### PR TITLE
Week 1 final fixes

### PR DESCRIPTION
**Pull request by joshhting**
- [x] Add Matthew Woodward's introduction slide
- [ ] Topic slides revisions
- [x] Git slides revisions

### CODE REVIEW

**Review by jgkamat — DISMISSED** [review]
LGTM so far

File: slides/week1/week1.org [hunk 1]
```diff
@@ -137,6 +152,9 @@ Run some demos in your terminal! Ideally you should use each of these commands a
 | =history= | Shows your command history |
 
 * Basic hotkeys
+#+BEGIN_NOTES
+Copy/paste are generally ctrl+shift+ctrl+shift+v but varies by shell.
```
Comment (jgkamat) [inline slides/week1/week1.org]: ctrl+shift+{c,v}

**Review by jgkamat — DISMISSED** [review]
LGTM

File: slides/week1/week1.org (hunk 1 above)
Comment (kberzinch) [reply slides/week1/week1.org]: oh geez how did that happen

File: slides/week1/git.org [hunk 2]
```diff
@@ -42,12 +46,15 @@
...
 
 * Install Git on your computer
-- Windows/Linux: https://git-scm.com/downloads
-- If on Mac OS, install Homebrew first from https://brew.sh/
+- Windows: https://git-scm.com/downloads
+- Mac OS:
+    - Install Homebrew first from https://brew.sh/
     - Then run: ~brew install git~
+- Ubuntu:
+    - Run ~sudo apt-get install git~
```
Comment (JasonGibson274) [inline slides/week1/git.org]: do we want to use apt instead of apt-get

File: slides/week1/week1.org [hunk 3]
```diff
@@ -126,8 +137,12 @@ Run some demos in your terminal! Ideally you should use each of these commands a
 | =mkdir= | Make a new directory |
 | =rm= | Remove a file |
 | =rmdir= | Remove a directory |
+| =echo= | Echoes back anything you pass |
```
Comment (JasonGibson274) [inline slides/week1/week1.org]: Using the word in the definition. Perhaps something like 

 echo - Write arguments to the standard output

**Review by JasonGibson274 — CHANGES_REQUESTED** [review]
overall very good just a small change

**Comment by joshhting** [comment]
If no complaints I'm going to merge this before our first meeting at 6 today.

**Review by JasonGibson274 — APPROVED** [review]

**Review by jgkamat — APPROVED** [review]
lets get this in more than an hour before pleaseeee

**Comment by jgkamat** [comment]
Melpa is having some infrastructure problems, which I don't think will affect us, but I want to merge these early so I can do a manual deploy if needed. We can follow up with another PR if anyone is :-1:

**Pull request merged**